src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/EnvelopeHandler.java

```diff
@@ -66,6 +74,20 @@ public void handleEnvelope(Envelope envelope) {
         }
     }
 
+    private void isJurisdictionConfigured(Envelope envelope) {
```

Comment:
'is' prefix suggests boolean return type. 'checkIfJurisdictionIsConfigured' looks better in this case.